Automattic/jetpack
# Add/icons for all product cards on dashboard

## Proposed changes:

* Ensure every product card on the Dashboard has an icon when inactive. Currently several are missing

### Other information:


## Jetpack product discussion

P2: pbNhbs-bHl-p2

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Checkout this branch via the Jetpack Beta plugin or your local dev environment
2. With a site connection but no user connection, go to `/wp-admin/admin.php?page=jetpack#/dashboard` and ensure that all cards relating to products have an icon
![image](https://github.com/user-attachments/assets/d0b84310-abb4-4c87-8d08-1df901a95152)
![image](https://github.com/user-attachments/assets/ccdfa142-9a4b-489d-ac5c-cfdd328b2c81)
3. Connect your user account
4. Go back to `/wp-admin/admin.php?page=jetpack#/dashboard` and ensure all products that need a plan or a standalone plugin installed still have an icon
![image](https://github.com/user-attachments/assets/8c82f962-aa3b-46f7-9a19-a27742e8ad82)
5. Try purchasing a few plans or installing a few plugins to activate products and make sure the cards look good still. Rule of thumb I used is if there is a CTA, we still show the icon
![image](https://github.com/user-attachments/assets/8934ee67-3535-46b6-b421-de54026cf58d)
![image](https://github.com/user-attachments/assets/d7382ce6-43af-4d5d-9d73-530e00b29845)
